src/cljs/main/broadfcui/page/test_login.cljs

```diff
@@ -0,0 +1,50 @@
+(ns broadfcui.page.test-login
+  (:require
+   [dmohs.react :as react]
+   [broadfcui.components.buttons :as buttons]
+   [broadfcui.components.foundation-tooltip :as tooltip]
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.input :as input]
+   [broadfcui.common.style :as style]
+   [broadfcui.nav :as nav]
+   [broadfcui.utils :as utils]
+   [broadfcui.utils.user :as user]
+   ))
+
+(react/defc Page
+  {:render
+   (fn [{:keys [this refs]}]
+    [:div {}
+      [:div {}
+        "Token:"
+        [input/TextField {:ref "force-token"
+                          :autoFocus true
+                          :type "password"
+                          :data-test-id "force-token"
+                          :style {:width "85%" :margin "0 1rem"}}]
+        [buttons/Button {:text "Force Sign-In" :onClick
+        (fn [e]
+          (.preventDefault e)
```

Comment:
This doesn't seem to be needed?

Reply:
empirically, it threw errors if I did not include this

Reply:
Huh! Weird, fair enough.